CSharpMath.Tests/PreTypesetting/MathListBuilderTest.cs

```diff
@@ -5,10 +5,13 @@
 using CSharpMath.Atoms;
 using CSharpMath.Interfaces;
 using CSharpMath.Enumerations;
+using System.Text.RegularExpressions;
 
 namespace CSharpMath.Tests {
 
   public class MathListBuilderTest {
+    const string RemoveWhiteSpacesPattern = "\\s*";

```

Comment:
Better to construct a readonly Regex instance instead of a string constant. The meaning is conveyed better.

Reply:
Unresolved